doc/api/cli.md

```diff
@@ -357,6 +357,48 @@ Error: Access to this API has been restricted
 }
 ```
 
+### `--allow-net-dns`
```

Comment:
I think instead of being a _true_/_false_ option, we should accept a list of DNS allowed to lookup.